client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -74,8 +76,14 @@ function useCommandNavigation() {
 						is_wp_admin: url.includes( '/wp-admin' ),
 					} )
 				);
+
 				close();
-				navigate( url );
+
+				if ( openInNewTab ) {
```

Comment:
Nice, I like this much better in terms of user experience

Reply:
I think the only part that I found a bit strange is the behavior for private sites that requires you to log in when you view a private site for the first time in a new tab. It both makes sense because it is a new tab but also a bit strange as I would expect to be already logged in. It happens though only for the first time, once you login and attempt the command the second time, the site's content is shown.

Reply:
I think I can use the `logmein` function...

Reply:
Done, @katinthehatsite. Please try again!

Reply:
Much better for the user experience 👍